src/v/datalake/translation/scheduling.cc

```diff
@@ -80,14 +80,15 @@ class default_reservations_tracker : public reservations_tracker {
      * then waiting for sufficient disk resources to become available.
      */
     ss::future<reservation>
-    reserve_disk(size_t bytes, ss::abort_source& as) override {
-        auto opt_units = ss::try_get_units(_available_disk, bytes);
+    reserve_disk(size_t, ss::abort_source& as) override {
+        const auto nr = _reservation_block_size;
```

Comment:
Suggest using a different variable for this even if it will be assigned the same value. Could prevent accidental bugs when the value is tuned in the future. Also, will ensure the constant and the thinking behind it is properly documented in the context of disk reservations.

Or, is there a hidden meaning in this being equal to memory reservation?

Reply:
Yeh good question. I changed the variable name and added an explanation for its current form here https://github.com/redpanda-data/redpanda/pull/25766/commits/f1535aafafbb1d41f188b5cd96eafee3e5a0b862. Tl;dr i'm not sure it deserves a formal configuration value.